src/httpc.c

```diff
@@ -464,6 +464,36 @@ httpc_set_accept_encoding(struct httpc_request *req, const char *encoding)
 #endif
 }
 
+int
+httpc_set_http_version(struct httpc_request *req, const char *version)
+{
+	if (strcmp(version, "1.0") == 0){
+		curl_easy_setopt(req->curl_request.easy, 
+			CURL_HTTP_VERSION_1_0, version);
```

Comment:
The option is `CURLOPT_HTTP_VERSION`, its value is `CURL_VERSION_1_0`.

```c
curl_easy_setopt(req->curl_request.easy, CURLOPT_HTTP_VERSION,
		 CURL_VERSION_1_0);
```

Reply:
Fixed with 
'''C
curl_easy_setopt(req->curl_request.easy,
        CURLOPT_HTTP_VERSION, (long)CURL_HTTP_VERSION_1_0); 
'''
according to https://curl.se/libcurl/c/CURLOPT_HTTP_VERSION.html